docs/cmdlet-example/command-line-simple-example.md

```diff
@@ -171,7 +171,7 @@ can't find the `netstandard.dll` "implementation" assembly for the version of th
 
 ### The fix for missing netstandard.dll
```

Comment:
There are stale references to 2.0 in this section.  Two that stand out are

* "Install the .NET Core SDK 2.0 for Windows, if it isn't already installed."
This actually links to the .NET Core 2.1 page.  Perhaps updating the link label to also use 2.x

* Copy-Item 'C:\Program Files\dotnet\sdk\2.0.0\Microsoft...
A note about the version number varying might help so users arent' blindly copy/pasting the text and getting errors.